docs/process-services-cloud/services/people-cloud.service.md

```diff
@@ -0,0 +1,44 @@
+---
+Title: People Cloud service
+Added: v3.0.0
+Status: Active
+Last reviewed: 2019-07-12
+---
```

Comment:
This information is wrong. Please modify the Added version and the Last reviewed

Reply:
Modified. Thanks